plugins/flatpak/gs-plugin-flatpak.c

```diff
@@ -130,27 +130,50 @@ gs_plugin_flatpak_add_installation (GsPlugin *plugin,
 	return TRUE;
 }
 
+static void
+gs_plugin_flatpak_report_warning (GsPlugin *plugin,
+				  GError **local_error)
+{
+	g_autoptr(GsPluginEvent) event = gs_plugin_event_new ();
+	gs_flatpak_error_convert (local_error);
+	gs_plugin_event_set_error (event, *local_error);
+	gs_plugin_event_add_flag (event,
+				  GS_PLUGIN_EVENT_FLAG_WARNING);
+	gs_plugin_report_event (plugin, event);
+}
+
 gboolean
 gs_plugin_setup (GsPlugin *plugin, GCancellable *cancellable, GError **error)
 {
 	GsPluginData *priv = gs_plugin_get_data (plugin);
+	guint num_installations_added = 0;
```

Comment:
Why do you need to count this separately instead of just looking at the length of `priv->flatpaks`?

Reply:
You're right. I will change that.